AK/Platform.h

```diff
@@ -27,6 +27,20 @@
 #    define AK_OS_BSD_GENERIC
 #endif
 
+// clang-format off
```

Comment:
What happens if we don't do that?

Reply:
it ruins the token paste. 

`#    define __STR(x) #x` 

turns into

`#    define __STR(x) #    x`

Reply:
Yikes! Filed https://github.com/llvm/llvm-project/issues/56602 for this.

Reply:
thanks, I've referenced your llvm issue above the suppression.